src/main/java/ch/njol/skript/variables/Variables.java

```diff
@@ -144,6 +144,13 @@ public Class<? extends ConfigurationSerializable> getClass(@NonNull String id) {
 	 */
 	static final List<VariablesStorage> STORAGES = new ArrayList<>();
 
+	/**
+	 * @return a copy of the list of variable storage handlers
+	 */
+	public static List<VariablesStorage> getStores() {
+		return Collections.unmodifiableList(STORAGES);
```

Comment:
Might be worth annotating this as Unmodifiable or Unmodifiable View. I don't recall whether this method creates a copy or a view.